### PR TITLE
synology-surveillance-station: Add version 1.2.11-2181

### PR DESCRIPTION
**Pull request by lsibilla**
Add Synology Surveillance Station client

### CODE REVIEW

File: bucket/synology-surveillance-station.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    "shortcuts": [
+        [
+            "bin\\SynologySurveillanceStationClient.exe",
+            "Synology Surveillance Station Client"
+        ]
+    ],
+    "checkver": {
+        "url": "https://archive.synology.com/download/Utility/SurveillanceStationClient",
+        "regex": "<a href=\"/download/Utility/SurveillanceStationClient/(?<version>[0-9\\.-]*)\" rel=\"noreferrer noopener\">(?<displayVersion>[0-9\\.-]*)</a>",
```
Comment (issaclin32) [inline bucket/synology-surveillance-station.json:21]:
```suggestion
        "regex": "/download/Utility/SurveillanceStationClient/([\\d.-]+)"
```
I think `/download/Utility/SurveillanceStationClient/...` is enough to filter out the information. There is no need for the additional `rel=....`.

File: bucket/synology-surveillance-station.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "replace": "${version}"
```
Comment (issaclin32) [inline bucket/synology-surveillance-station.json:22]:
```suggestion

```
*Scoop* takes the 1st match group as **version**. Therefore this is not needed.

File: bucket/synology-surveillance-station.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+    "version": "1.2.11-2181",
```
Comment (issaclin32) [inline bucket/synology-surveillance-station.json:2]:
```suggestion
    "version": "1.2.11-2181",
    "description": "Synology Surveillance Station client",
    "homepage": "https://www.synology.com/en-us/surveillance",
```
`description` and `homepage` is required for official buckets.

Comment (lsibilla) [reply bucket/synology-surveillance-station.json]: It seems I forgot a push. This one is fixed now.

File: bucket/synology-surveillance-station.json (hunk 1 above)
Comment (lsibilla) [reply bucket/synology-surveillance-station.json]: @issaclin32 I pushed this change.

File: bucket/synology-surveillance-station.json (hunk 2 above)
Comment (lsibilla) [reply bucket/synology-surveillance-station.json]: @issaclin32 I pushed this change

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**